src/Generator.php

```diff
@@ -13,7 +13,7 @@
 use Nette\Http;
 use Tracy\Debugger;
 
-class Generator
+final class Generator
```

Comment:
Proc final?

Reply:
Proc ne? :D

Reply:
Jen se ptam :-) Klidne muze byt, ale u knihoven jsem s final opatrny. Kdyz to chce u sebe nekdo pretizit.

Reply:
OK.